requirements.txt

```diff
@@ -17,20 +17,20 @@ eventlet==0.25.2
 
 awscli-cwlogs>=1.4,<1.5
 
-gds-metrics==0.2.0
+gds-metrics==0.2.4
 
 git+https://github.com/alphagov/notifications-utils.git@36.9.3#egg=notifications-utils==36.9.3
 
 ## The following requirements were added by pip freeze:
-awscli==1.18.95
+awscli==1.18.110
 bleach==3.1.4
 blinker==1.4
-botocore==1.17.18
+botocore==1.17.33
 certifi==2020.6.20
 chardet==3.0.4
 click==7.1.2
 colorama==0.4.3
-dnspython==1.16.0
+dnspython==2.0.0
```

Comment:
This is not compatible with eventlet 0.25.2. I found that out the hard way last week when I broke preview 😬